spec/models/conference/importer_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
```

Comment:
Please stop ignoring :-)
I spent a lot of time solving the wrong problem 🙈 . Sorry that it took so long. 
In the specs, I use the conferences from the test file, and now I am not sure if that is the right level of abstraction for the specs.